test/parallel/test-fs-promises-readfile.js

```diff
@@ -72,7 +72,7 @@ async function validateWrongSignalParam() {
 async function validateZeroByteLiar() {
   const originalFStat = fsBinding.fstat;
   fsBinding.fstat = common.mustCall(
-    () => (/* stat fields */ [0, 1, 2, 3, 4, 5, 6, 7, 0 /* size */])
+    async () => (/* stat fields */ [0, 1, 2, 3, 4, 5, 6, 7, 0 /* size */])
   );
```

Comment:
This now needs to be an async function otherwise `binding.fstat(/* */).catch` will fail.